examples/starknet-react-next/src/pages/index.tsx

```diff
@@ -8,19 +10,27 @@ import { useCounterContract } from '~/hooks/counter'
 const Home: NextPage = () => {
   const { contract: counter } = useCounterContract()
 
-  const { data: counterValue } = useStarknetCall({ contract: counter, method: 'counter', args: {} })
+  const { data: counterResult } = useStarknetCall({
+    contract: counter,
+    method: 'counter',
+    args: [],
+  })
+
+  const counterValue = useMemo(() => {
+    if (counterResult && counterResult.length > 0) {
+      const value = toBN(counterResult[0])
+      return value.toString()
+    }
+  }, [counterResult])
 
   return (
     <div>
       <h2>Wallet</h2>
       <ConnectWallet />
       <h2>Counter Contract</h2>
-      <p>Address: {counter?.connectedTo}</p>
-      <p>Value: {counterValue?.count}</p>
-      <p>
-        Count:{' '}
-        {typeof counterValue?.count === 'string' && hexadecimalToDecimal(counterValue?.count)}
-      </p>
+      <p>Address: {counter?.address}</p>
+      <p>Value: {counterValue}</p>
+      <p>Count: {typeof counterValue === 'string' && hexadecimalToDecimal(counterValue)}</p>
```

Comment:
Hm should the `toBN` above manage the hex conversion for us?

Reply:
I will fix this.